presto-geospatial/pom.xml

```diff
@@ -67,16 +72,10 @@
         </dependency>
 
         <!-- for testing -->
-        <dependency>
-            <groupId>com.facebook.presto</groupId>
-            <artifactId>presto-parser</artifactId>
-            <scope>test</scope>
-        </dependency>
-
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
-            <scope>test</scope>
+            <scope>compile</scope>
```

Comment:
@tdcmeehan `compile` is the default scope, hence, it doesn't need to be specified explicitly - https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Dependency_Scope

Why do you need all of these dependencies?